tools/mason/MasonHelp.chpl

```diff
@@ -139,3 +141,18 @@ proc masonEnvHelp() {
   writeln("Environment variables set by the user will be printed with an");
   writeln("asterisk at the end of the line.");
 }
+
+proc masonTestHelp() {
+  writeln("Run test files located within target/debug/tests");
```

Comment:
I think it'd be nice to stay consistent with `test/` or `tests/` for both the top-level source directory and the build directories.